terraform_egress_pub/scripts/egress_pub.py

```diff
@@ -54,23 +55,29 @@
             '64.39.99.98/31', '64.39.99.100/32', '64.39.99.131/32',
             '64.39.99.132/30', '64.39.99.136/29', '64.39.99.144/31',
         ),
-        'description': 'This file contains a consolidated list of all the IP addresses that NCATS is currently using for external scanning.'
+        'description': 'This file contains a consolidated list of all the IP addresses that VM is currently using for external scanning.'
     }, {
         'filename': 'cyhy.txt',
         'app_regex': re.compile('(Manual )?Cyber Hygiene$'),
         'static_ips': (
             '100.27.42.128/25', '64.69.57.0/24',
-            '64.39.99.19/32', '64.39.99.20/30', '64.39.99.24/31',
-            '64.39.99.98/31', '64.39.99.100/32', '64.39.99.131/32',
-            '64.39.99.132/30', '64.39.99.136/29', '64.39.99.144/31',
         ),
         'description': 'This file contains a list of all IPs used for Cyber Hygiene scanning.'
     }, {
         'filename': 'pca.txt',
         'app_regex': re.compile('Phishing Campaign Assessment$'),
         'static_ips': (),
         'description': 'This file contains a list of all IPs used for Phishing Campaign Assessments'
-    }]
+    }, {
+        'filename': 'was.txt',
+        'app_regex': re.compile('Web Application Scanning$'),
+        'static_ips': (
+            '64.39.99.19/32', '64.39.99.20/30', '64.39.99.24/31',
+            '64.39.99.98/31', '64.39.99.100/32', '64.39.99.131/32',
+            '64.39.99.132/30', '64.39.99.136/29', '64.39.99.144/31',
+        ),
```

Comment:
Since it is now duplicated, perhaps this tuple of strings should be broken out into it's own variable, something like `qualys_was_ips`?

Reply:
See commit 54a3835.

Reply:
Ah, I wondered about that as well. Since it was duplicated before I assumed that was intentional.